Spec/Auth.swift

```diff
@@ -1617,7 +1617,7 @@ class Auth : QuickSpec {
         describe("authorise") {
 
             // RSA10a
-            it("should create a token if needed and use it") {
+            it("should reuse the valid token and ensure Token Auth is used for all future requests") {
```

Comment:
Nope, it should not reused the valid token, but should create a token and ensure Token Auth...